src/test/java/com/epam/tc/hw3/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,64 @@
+package com.epam.tc.hw3.ex1;
+
+import com.epam.tc.hw3.AbstractChromeTest;
+import java.io.IOException;
+import java.util.List;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import pages.MainPage;
+
+import static pages.MainPage.*;
+
+public class Exercise1Test extends AbstractChromeTest {
+
+
```

Comment:
Let's say final also. And when you declare constants it's always better to have their names in uppercase. Just a convention.